packages/xo-server/src/xapi/index.js

```diff
@@ -1431,7 +1431,21 @@ export default class Xapi extends XapiBase {
     return /* await */ this._snapshotVm(this.getObject(vmId), nameLabel)
   }
 
-  async _startVm(vm, host, force) {
+  async _startVm(vm, host, force, ignoreMacAddressesCheck = false) {
```

Comment:
- use named options
- IMHO, `force` should disable all checks
- `bypassMacAdressesCheck` would be better aligned with existing code: https://github.com/vatesfr/xen-orchestra/blob/d191ca54ad7b864d8569ae8c15f49fd657978590/%40xen-orchestra/backups/_VmBackup.js#L120

```suggestion
  async _startVm(vm, host, {
    force = false,
    bypassMacAddressesCheck = force,
  } = {}) {
```